### DISCUSSION TITLE
intravenous drip 2-compartment model

### DISCUSSION THREAD

**Discussion by Mannny-PK** [LAPKB/Pmetrics#78] [Q&A] [answered]
Hello!  I developed a intravenous drip 2-compartment model by Pmetrics 1.9.7,but  I am confused by a question: the intravenous drip model should not include the parameter Ka, why does the second variable in the model include Ka?

The following is displayed when the model is run
Model solver mode: Algebraic
Number of compartments: 2
Primary Variables: Ke, KCP, KPC, V, Vp
Covariates in data file:  wt, scr, alb, asian, gender, age
Covariates used in model file:  WT, SCR, ALB, ASIAN, AGE, GENDER
Secondary Variables: KA
Model conditions: no bioavailability term defined, initial conditions are zero, no lag term defined
Number of cycles to run: 100

**Comment by mnneely** [comment, marked as answer]
Hi @Manny-PK,

In your case KA is set to zero. In order to choose the right model equations, Pmetrics looks first for a #DIFF block containing differential equations that define the structural model. If that block is not in the model, Pmetrics next looks for Ke and V. If it finds those in #PRI or #SEC blocks, it continues looking, now for KA, KCP, KPC. Any of these that it does not find are set to zero. In this way, it can choose the correct structural equations to define the model.

On Jul 10, 2022, at 2:19 AM, Mannny-PK ***@***.***> wrote:

﻿

Hello! I developed a intravenous drip 2-compartment model by Pmetrics 1.9.7,but I am confused by a question: the intravenous drip model should not include the parameter Ka, why does the second variable in the model include Ka?

The following is displayed when the model is run
Model solver mode: Algebraic
Number of compartments: 2
Primary Variables: Ke, KCP, KPC, V, Vp
Covariates in data file: wt, scr, alb, asian, gender, age
Covariates used in model file: WT, SCR, ALB, ASIAN, AGE, GENDER
Secondary Variables: KA
Model conditions: no bioavailability term defined, initial conditions are zero, no lag term defined
Number of cycles to run: 100

**Reply by Mannny-PK** [reply]
Thank you very much for your answer, it helped me a lot.

**Comment by Mannny-PK** [comment]
I see, thank you very much for your answer!
